packages/console/src/pages/Applications/index.tsx

```diff
@@ -89,69 +85,50 @@ const Applications = () => {
           />
         </Modal>
       </div>
-      <StickyHeaderTable
+      <Table
         className={resourcesStyles.table}
-        header={
-          <thead>
-            <tr>
-              <th>{t('applications.application_name')}</th>
-              <th>{t('applications.app_id')}</th>
-            </tr>
-          </thead>
-        }
-        colGroup={
-          <colgroup>
-            <col className={styles.applicationName} />
-            <col />
-          </colgroup>
-        }
-      >
-        <tbody>
-          {!data && error && (
-            <TableError
-              columns={2}
-              content={error.body?.message ?? error.message}
-              onRetry={async () => mutate(undefined, true)}
-            />
-          )}
-          {isLoading && <TableLoading columns={2} />}
-          {applications?.length === 0 && (
-            <TableEmpty columns={2}>
-              <Button
-                title="applications.create"
-                type="outline"
-                onClick={() => {
-                  navigate({
-                    pathname: createApplicationPathname,
-                    search,
-                  });
-                }}
+        data={applications}
```

Comment:
How about calling it `rows` or `rowData`?

Reply:
Aha, good idea, let me try